tools/ci_build/github/azure-pipelines/linux-ci-pipeline.yml

```diff
@@ -87,9 +98,20 @@ jobs:
   - task: CmdLine@2
     displayName: 'Run Debug unit tests'
     inputs:
-      scriptPath: $(Build.SourcesDirectory)/tools/ci_build/build.py
-      arguments: --build_dir $(Build.BinariesDirectory) --cmake_generator Ninja --config Debug --test --skip_submodule_sync --build_shared_lib --parallel --build_wheel --enable_onnx_tests --enable_transformers_tool_test --build_nodejs --ctest_path ""
-      workingDirectory: /tmp
+      script: |
+        python3 $(Build.SourcesDirectory)/tools/ci_build/build.py \
```

Comment:
>python3 $ [](http://example.com/codeflow?start=8&length=9)

How does this add, since the command like appears to be the same? Was is skipped before bc python was not there?

Reply:
I put wrong task name there. I was about to use "task: PythonScript@0"

Reply:
python3 may be unnecessary, just made it consistent with the release unit tests one. as @snnn mentioned, the inputs were getting ignored because the properties are not recognized for the CmdLine task